test/units/jwt_sessions/test_refresh_token.rb

```diff
@@ -7,6 +7,7 @@ class TestRefreshToken < Minitest::Test
    attr_reader :csrf, :token, :access_uid
 
   def setup
+    JWTSessions.token_store = :redis
```

Comment:
As a side note: there're kind of integration tests, defined within sinatra/rails dummy apps.
It'd be great to run them with memory store (and generic unit tests also). 
Can we extend the Rakefile with rake tasks to run the same test suites, but using memory store as default?

Reply:
working on it